cms/djangoapps/contentstore/management/commands/add_instructor_ids.py

```diff
@@ -0,0 +1,59 @@
+"""
+Command to add UUID for Instructor.
+"""
+
+import logging
+import uuid
+
+from django.core.management.base import BaseCommand
+from xmodule.modulestore.django import modulestore
+
+from openedx.core.djangoapps.models.course_details import CourseDetails
+from django.contrib.auth.models import User
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Example usage:
+        $ ./manage.py cms add_instructor_ids --settings=devstack
+    """
+    help = 'Generates and stores course overview for one or more courses.'
+
+    def handle(self, *args, **options):
+
+        course_keys = [course.id for course in modulestore().get_course_summaries()]
```

Comment:
We should be able to provide the command with an organization or list of course keys for which we want the uuid populated instead of iterating over all courses.